nativesintech/MMIWG2S
# Add Share page.

To Test:
Start up the app
Tap the Capture button
Tap the Share button
The modal should slide up :)

Everything works, but the share with us part. Still need the networking code merged. But this has most of the functionality we need.
